boilerplate/app/components/Button.tsx

```diff
@@ -105,7 +117,7 @@ export function Button(props: ButtonProps) {
   }
 
   return (
-    <Pressable style={$viewStyle} accessibilityRole="button" {...rest}>
+    <Pressable style={$viewStyle} accessibilityRole="button" {...rest} disabled={disabled}>
```

Comment:
I'm not an accessibility expert, but do we need a `accessibilityState={{disabled}}` here (similar to the `Toggle` component)?

Reply:
Great catch! Fixed in https://github.com/infinitered/ignite/pull/2593/commits/e6b72d46c1985c86ed4831e8251afe799c247549